fpc_imp_yoshino.c

```diff
@@ -48,38 +48,39 @@ typedef struct {
 
 err_t fpc_deep_sleep(fpc_imp_data_t *data);
 
+static const char *error_strings[] = {
+#ifdef USE_FPC_NILE
+    "FPC_ERROR_TEMPLATE_CORRUPTED",
+    "FPC_ERROR_DB",
+    "FPC_ERROR_CRYPTO",
+    "UNKNOWN_ERROR",
+    "UNKNOWN_ERROR",
+#endif
+    "FPC_ERROR_PN",
+    "FPC_ERROR_RESET_HARDWARE",
+    "FPC_ERROR_NOT_INITIALIZED",
+    "FPC_ERROR_CONFIG",
+    "FPC_ERROR_HARDWARE",
+    "FPC_ERROR_NOENTITY",
+    "FPC_ERROR_CANCELLED",
+    "FPC_ERROR_IO",
+    "FPC_ERROR_NOSPACE",
+    "FPC_ERROR_COMM",
+    "FPC_ERROR_ALLOC"
+    "FPC_ERROR_TIMEDOUT",
+    "FPC_ERROR_INPUT",
+};
```

Comment:
please add an FPC_ERROR_MAX entry to avoid going out of bounds

Reply:
Done, but in a different way.